Cabal/Cabal.cabal

```diff
@@ -459,3 +485,21 @@ test-suite package-tests
   ghc-options: -Wall -rtsopts
   default-extensions: CPP
   default-language: Haskell98
+
+test-suite parser-tests
+  if !flag(parsec-debug)
+    buildable: False
```

Comment:
When I `cabal new-build Cabal` in our current tooling, do you know if the solver toggles `parsec-debug` on or off by  default? If it is off by default I am a bit annoyed, because then if I want to run parser tests I have to rebuild the library.